### ISSUE TITLE
train on COCO but precision,recall and cls_acc Value has not changed it always 0

### ISSUE THREAD

**Issue by MRZWD** [eriklindernoren/PyTorch-YOLOv3#390]
I tried this command
python3 train.py --data_config config/coco.data  --pretrained_weights weights/darknet53.conv.74

but get some worry. why the recall,precision,cls_acc values is always 0.
![image](https://user-images.githubusercontent.com/25874451/71353285-8214db00-25b3-11ea-9835-8a051f270b56.png)

**Comment by MOHAMEDELDAKDOUKY** [comment]
Hello! 
Have you figured out why such case exists? I got the same problem training on a subset of 16 coco images.

**Comment by xiaoyuzhu666** [comment]
I got the same problem.

**Comment by Chauncy-Cai** [comment]
I have a similar problem, my cls_acc is not 0% but just around 15~20%. The same thing is my recall and precision remain 0 after 4k steps, while loss seems not change (about 16~20)
